src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -20,6 +22,13 @@ Loader {
         height: 40
     }
 
+    property StatusIdenticonRingSettings ringSettings: StatusIdenticonRingSettings {
+        ringSpec: undefined
```

Comment:
Just a nitpic: There's no need to define it as `undefined` here. Leave it out and it'll be undefined as well.